packages/fiori-elements-writer/src/types.ts

```diff
@@ -103,7 +103,8 @@ export interface FioriElementsApp<T> extends Ui5App {
     app: FioriApp;
     appOptions: Partial<AppOptions> & {
         /**
-         * Generate OPA based tests, if applicable to the specified template.
+         * Generate OPA based tests, if applicable to the specified template and OData Version is 4.
+         * The tests may not work if service metadata is not provided.
```

Comment:
Not sure I'd bother with the change to the comments. This should be extended to support v2 tests also (in a separate TBI).